contracts/modules/STO/USDTieredSTO.sol

```diff
@@ -217,10 +217,10 @@ contract USDTieredSTO is ISTO, ReentrancyGuard {
         require(_ratePerTierDiscountPoly.length == _tokensPerTierTotal.length, "Mismatch b/w discount rates & tokens / tier");
         require(_tokensPerTierDiscountPoly.length == _tokensPerTierTotal.length, "Mismatch b/w discount tokens / tier & tokens / tier");
         for (uint8 i = 0; i < _ratePerTier.length; i++) {
-            require(_ratePerTier[i] > 0, "Rate > 0");
-            require(_tokensPerTierTotal[i] > 0, "Tokens per tier > 0");
-            require(_tokensPerTierDiscountPoly[i] <= _tokensPerTierTotal[i], "Discounted tokens / tier <= to tokens / tier");
-            require(_ratePerTierDiscountPoly[i] <= _ratePerTier[i], "Discounted rate / tier <= rate / tier");
+            require(_ratePerTier[i] > 0, "Rate is not larger than 0");
+            require(_tokensPerTierTotal[i] > 0, "Tokens per tier is not larger than 0");
+            require(_tokensPerTierDiscountPoly[i] <= _tokensPerTierTotal[i], "Discounted tokens / tier is larger than tokens / tier");
+            require(_ratePerTierDiscountPoly[i] <= _ratePerTier[i], "Discounted rate / tier is larger than rate / tier");
```

Comment:
This contract is on verge of going over the limit as well. We might have to revert these changes if it does end up going over the limit. changes are good otherwise.